utils/build_profiler.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+if [ "$#" -ne 1 ]; then
+    echo "build_profiler.sh requires one argument - product_string"
+    exit 1
+fi
+
+product_string="$1"
+
+# Create and change to .build_profiling dir 
+if [ ! -d ".build_profiling" ]; then
+    mkdir .build_profiling
```

Comment:
It is a good idea to have a `die` function defined in your script, and then just to write

```
mkdir .build_profiling || die "Creating the .build_profiling directory failed"
```